src/test/java/edu/cornell/kfs/sys/service/impl/AwsSecretServiceImplIntegrationTest.java

```diff
@@ -16,9 +19,10 @@
 import com.fasterxml.jackson.databind.JsonMappingException;
 
 import edu.cornell.kfs.sys.service.impl.fixture.AwsSecretPojo;
-import net.bull.javamelody.internal.common.LOG;
 
 class AwsSecretServiceImplIntegrationTest {
+    private static final Logger LOG = LogManager.getLogger(AwsSecretServiceImplIntegrationTest.class);
```

Comment:
Another LogManager.getLogger() call that can be simplified to use the no-arg method.